packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, useState } from 'react';
+import i18next from 'i18next';
+import { ColorPalette, DEFAULT_LOCALE } from '@vse-bude/shared';
+import { notification } from '~/services/services';
+import { TouchableOpacity, Text, View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const LanguageButtons: FC = () => {
+  const [currentLang, setCurrentLang] = useState(i18next.language);
+  const handleChangeLanguage = (lang: string) => {
+    i18next.changeLanguage(lang, (err) => {
+      setCurrentLang(lang);
+      if (err) {
+        notification.error(err);
+      }
+    });
+  };
+  const isUa = currentLang == DEFAULT_LOCALE;
+
+  return (
+    <View
+      style={[
+        globalStyles.mr4,
+        globalStyles.flexDirectionRow,
+        globalStyles.justifyContentEnd,
+      ]}
+    >
+      <TouchableOpacity
```

Comment:
Try to refactor it to something like this:
```
{i18n.languages.map(lang => {
  const isCurrent = i18n.resolvedLanguage === lang;

  return (
   <TouchableOpacity style={[styles.languageButton, isCurrent && styles.current]}>
     <Text>{lang}</Text>
   </TouchableOpacity>
  );
})}
```

Reply:
done